WIP-extension.md

```diff
@@ -1,7 +1,33 @@
 Q:
 
-- [ ] Is FE aware of current crowdfunding phase?
+- [x] Is FE aware of current crowdfunding phase? _Yes_
 
+Flow to determine whether a prolong `packageOption` is returned.
+
+Input:
+- A membership
+- A prolong packageOption, to check against
+
+a)  Does user own membership, or pledge membership (ABO_GIVE)
+    - If false, end and don't return package option
+b)  Is membership active
+    - If false, end
```

Comment:
This would mean that memberships that have ended because they were not extended, need to be replaced by a new membership right? Should we not make it possible to reactive by extending?

Reply:
Thus far in my mind a `membership` should have `membershipPeriods` without a gap in between.

But the longer I think about it, it appears not to be a necessity.

Exploring potential downsides: Having gaps however would come at the expense of potentially more complex queries in database to retrieve statistical, financial data. If we were to allow `active: false` memberships to be extended, we'd probably need another check to prevent unclaimed "gift" memberships to be extended to eternity.

Sounds to me like a business decision.

Reply:
> Exploring potential downsides: Having gaps however would come at the expense of potentially more complex queries in database to retrieve statistical, financial data.
this is true. We need to do it anyways as we are already today extending (reactivateMembership) monthly memberships that have been deactivated before. The logic there is that everybody has max. 1 monthly memberhip. If it's canceld it's dormant (active=false) until it get's reactivated (new period, active=true)

> we'd probably need another check to prevent unclaimed "gift" memberships to be extended to eternity.
That should not be a problem as they are not active and don't have a membershipPeriod yet.